common/util_test.go

```diff
@@ -201,6 +201,22 @@ func TestVideoProfile_FormatExtension(t *testing.T) {
 		t.Error("Did not get expected error")
 	}
 }
+
+func TestVideoProfile_ProfileNameToValue(t *testing.T) {
+	assert := assert.New(t)
+	inp := []string{"", "h264baseline", "h264main", "h264high", "h264constrainedhigh"}
```

Comment:
Missing "none" which is included in the `EncoderProfileLookup` map in `EncoderProfileNameToValue()`?

Reply:
Fixed by adding a separate check in 4b17b6ca9b4e86f1e4c5d5648be235e0580fafc6 . Note that we can't add a new element to that list, since it matches exactly the number of profiles we have (it's intended to trigger a failure if we ever add a new profile to LPMS without corresponding support in go-livepeer)